### DISCUSSION TITLE
these authors contributed equally to this work

### DISCUSSION THREAD

**Discussion by castedo** [tskit-dev/msprime-1.0-paper#266] [General]
Somewhere along the journal publication process, the author list notes of: 

> Denotes shared first authorship, listed alphabetically
> Denotes shared senior authorship, listed alphabetically

got changed to

> These authors contributed equally to this work

I wanted to share my thoughts on this and let other chime in too. I'm extremely grateful for the work and responsibility that the Jereome has taken on for this paper. I don't want to make his work here more difficult thus I'm OK with whatever Jereome ends up deciding.

In my opinion, two of the top goals of research communication should be **accuracy** and **transparency**. Communicating what is true. This is in contrast to interests that conflict with the truth, such as "journal style", branding, making the journal "look good" or making the authors "look good", making more money, promotions, marketing, etc...

It is FALSE that the first authors "contributed equally to this work" and ditto for the senior authors. The text "listed alphabetically" in the preprint if 100% accurate and transparent and I can think of only reasons that are conflicts of interests to the truth.

It is worth noting that in [mathematics](https://www.ams.org/profession/leaders/culture/CultureStatement04.pdf), [high-energy particle physics, and economics](https://www.nature.com/articles/d41586-021-01574-y) authors are generally listed alphabetically.

Perhaps some lines should be added to the conflict of interest section that the journal has a conflict of interest with accurately describing the contributions of the authors?

**Comment by petrelharp** [comment]
I agree that the short phrase "contributed equally" is not 100% accurate (how could it be?), but I think what's generally understood by the phrase is "contributed roughly equivalently". Equivalently by which metric, you might ask? Hours of work? Lines of text? Number of edits? Essentialness to the paper? This isn't a very productive direction to go in, so I'm happy with the journal style, as stated. If we *do* want to be more precise, I think the thing to do would be to add something to the Acknowledgements explaining who worked on what (e.g., "PLR wrote code and text for the mutation section, and contributed to editing"). I'm not sure we shoudl try to do that at this late hour, however.

**Reply by castedo** [reply]
I agree that it would be much too late in the process to try and communicate more precise detail on relative degrees of contribution. Simply having four buckets as done in the preprint (first, middle, senior, corresponding) makes a lot of sense.

However, simply reverting to Jerome's original wording does not seem problematic. It was easy for someone at the journal to change it to a less accurate statement. It's seems like it would be easy to switch it back to Jerome's original more accurate wording.

And I'm not sure there is really any true late hour here. This is an online-only publication. My understanding is there is no actual physical printing on paper, not since 2010.

**Comment by castedo** [comment]
BTW, just in case there is confusion about "accurate" vs "precise". By _accurate_ I mean "closer to truth" whereas by _precise_ I mean "less room for interpretation".

So "These authors contributed equally to this work" **sounds** more _precise_ when in fact it is _inaccurate_. In contrast to "Denotes shared (first|senior) authorship, listed alphabetically" is 100% _accurate_ and could be considered less _precise_ if one is trying to judge levels of contributions.

To me this kind of language is like companies marketing their product saying "The #1 gizmo on the planet", "ranked #1 ..." or "9 out of 10 experts agree" etc... It's sound good, it's precise ("#1", "9 out of 10") and it's understood by many as not really meaning what it says. But in a scientific document for research communication why not just communicate truthfully?

**Comment by sgravel** [comment]
I agree with @petrelharp here. "These authors contributed equally" is not meant literally according to a quantitative metric, but is used as shorthand for something like "These authors contributed comparably and we feel like they should share the credit that is typically assumed based on authorship order". One of the main uses of such acknowledgements is so that scientists can get the corresponding credit, for example when they send a CV. If each article had a different, and perhaps more accurate statement, it would become much more challenging to interpret. 

I am not saying that this is a good system, just that there is value in using a journal- and field-standard phrase to acknowledge shared credit, and whose meaning is fairly well understood in the community. 

Even though there are conflicts of interest in assigning joint authorship, I don't think that these are more problematic than the usual COIs in choosing, e.g., authorship order. It doesn't mean that we should not try to address them as a field, but I do not see that trying to tackle this here and now would be productive. 

And I am also happy to defer to Jerome :)

**Comment by jeromekelleher** [comment]
I don't like the system either @castedo (I originally wanted to put everyone in alphabetically but decided this was even more unfair, see #2).

I don't think there's any point in pushing back against the journal here on this, it's far simpler to just follow the journal convention on this and in practice, everyone who matters understands the "code" anyway so there's no difference. We're not going to change the system here in this one paper, and we're just wasting time discussing how and why its broken.

The only question I had was whether all the first authors having an "equal" equal contribution to the senior authors equal contribution was pushing things a little bit too far, and just plain confusing. But my vote is to go with the journal's standard practise.

@sgravel, you're probably best placed to comment on this - is it standard to have the same "equal contribution" mark for first and senior authors in Genetics?

**Reply by castedo** [reply]
@jeromekelleher You've signed up for a big responsibility here and done a ton of awesome work. Totally understandable if you don't want to spend time pushing back on the journal and go against convention.

It's worth noting that I am neither a first author, nor senior author, nor sending my CV around, sooooo... this is basically none of my business. :sweat_smile: I'm really happy I could contribute to the code base and witness so much of this novel community process.

But I did want to take the opportunity to point out this systemic norm of stating something that is false.

**Reply by petrelharp** [reply]
I think it seems totally reasonable to ask Genetics if they'd be willing to have "joint first" and "joint last" instead.

**Reply by petrelharp** [reply]
Also note that in our manuscript we had this: 
![Screenshot from 2022-01-04 13-17-28](https://user-images.githubusercontent.com/1046249/148125074-a0eb286a-ce3c-4a8f-8b3b-f96d6e89ea3b.png)
... so their change seems totally reasonable. I'll ask them to change it to 
- "These authors contributed equally to this work, as joint first authors"
- "These authors contributed equally to this work, as joint last authors"

I guess I'm voting for not trying to say something less standard because that really is the standard phrase.

**Reply by sgravel** [reply]
I had never seen the same mark for both first and last, and I agree that it is confusing. It might just not happen all that often. In most scenarios, shared senior contribution would be indicated using "co-corresponding", but that would not work well for this paper. 

So I don't think that this is standard practice, and I am pretty sure that the journal will work with what we want to do (as per Ruth's email).

**Reply by jeromekelleher** [reply]
SGTM!
